MSAL/src/native_auth/public/state_machine/delegate/SignInDelegates.swift

```diff
@@ -110,6 +116,7 @@ public protocol SignInVerifyCodeDelegate {
     @MainActor func onSignInVerifyCodeError(error: VerifyCodeError, newState: SignInCodeRequiredState?)
 
     /// Notifies the delegate that the sign in operation completed successfully.
+    /// - Note: If a flow requires a code but this optional method is not implemented, then ``onSignInVerifyCodeError(error:newState:)`` will be called.
```

Comment:
Same here and in other places too, the doc comment is incorrect because it talks about a code being required.

Reply:
This is now fixed